local/components/kostyanvi/news/templates/news/component_epilog.php

```diff
@@ -0,0 +1,5 @@
+<? if (!defined("B_PROLOG_INCLUDED") || B_PROLOG_INCLUDED !== true) die();
+
+\Bitrix\Main\UI\Extension::load('ui.vue');
+\Bitrix\Main\UI\Extension::load('ui.vue.router');
+\Bitrix\Main\UI\Extension::load('ui.vue.vuex');
```

Comment:
Можно передавать массив из нескольких расширений:

```suggestion
\Bitrix\Main\UI\Extension::load([
  'ui.vue',
  'ui.vue.router',
  'ui.vue.vuex'
]);
```